_posts/2017-09-03-variables-in-inspector.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title:  Making Variables Visible in Inspector
+date:   2017-09-03
+categories: Tips&Tricks
+tags: inspector code beginner
+excerpt:
+mathjax: false
+author: BaconGamer88
+---
+
+Beginner programmers in Duality may have begun to create their scripts, only to wonder why they can't read/write variables in the 
+inspector. In Unity, you would declare the variable as public or put `[SerializeField]` in front of the variable. However, both approaches 
```

Comment:
If we mention serializing, why don't we explain Duality's `[DontSerialize]` attribute?